build/docker/start_kmesh.sh

```diff
@@ -1,11 +1,7 @@
 #!/bin/sh
 
-lsmod | grep kmesh > /dev/null
-if [ $? -ne 0 ]; then
-        cp kmesh.ko /lib/modules/`uname -r`
-        depmod -a
-        modprobe kmesh
-fi
+bash build.sh
```

Comment:
Why is it changed to build.sh? Is the original ko installation process unnecessary?